components/brave_shields/browser/ad_block_component_installer.cc

```diff
@@ -60,6 +60,17 @@ const char kAdBlockIosDefaultDatComponentBase64PublicKey[] =
     "5HcH/heRrB4MvrE1J76WF3fvZ03aHVcnlLtQeiNNOZ7VbBDXdie8Nomf/QswbBGa"
     "VwIDAQAB";
 
+const char kAdBlockIosComponentName[] = "Brave Ad Block Updater";
+const char kAdBlockIosComponentId[] = "iodkpdagapdfkphljnddpjlldadblomo";
+const char kAdBlockIosComponentBase64PublicKey[] =
```

Comment:
Can we make it more obvious that this component isn't iOS specific?

Reply:
Either that or find a way to share the component defined in the [ad_block_service.cc](https://github.com/brave/brave-core/blob/7963b1cb3df30d08bc1bb8144fe422f6b1f16fb6/components/brave_shields/browser/ad_block_service.cc#L44)

Reply:
This is all temporary and can be removed once we switch to the non-legacy version